docker/gcp_pudl_etl.sh

```diff
@@ -124,4 +134,5 @@ else
     notify_slack "failure"
 fi
 
+copy_outputs_to_gcs
```

Comment:
Do we want this to be the very last thing that happens? Or do we want to copy the outputs to GCS  before we cleanup the output directory for distribution? (removing Datasette's `metadata.yml`, the partitioned CEMS parquet files, etc.) so that we still have that information if we need it for debugging?  Seems like we probably want to do the gzipping before shoving it into the nightly build bucket.

Reply:
This retains current behavior in that we used to copy things to nightly-buckets as part of `vm_shutdown`. Happy to move this to earlier stage if we think it's a good idea.